services/docker-protonet-network.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Docker protonet bridge network initialization
+After=docker.service
+
+[Service]
+ExecStart=-/usr/bin/env docker network create --driver bridge protonet
+ExecStartPost=/bin/bash -c "docker network inspect protonet | jq '.[0].IPAM.Config[0].Gateway' --raw-output | cut -d '/' -f1 > /etc/protonet/system/dockerhost"
+KillMode=none
+Type=oneshot
```

Comment:
This unit won't be enabled on boot as it has no `[[Install]]` section. Is that the way it's supposed to be? @colszowka